servant-server/src/Servant/Server.hs

```diff
@@ -128,6 +131,11 @@ import           Servant.Server.UVerb
 
 -- * Implementing Servers
 
+-- All constraints on a context passed to serveWithContext.
+type ServerContext context =
```

Comment:
Could you turn this into a proper haddock docstring that concisely explains what this synonym is about?